### PR TITLE
Feature/replace compass with libsass

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- please check carefully replacement of @include boder-radius()
- will run csscomb in another pull request

- [x] responsive behaviour tested
- [x] browser and device tested
- [x] passes validation (html / bootstrap)
- [x] SEO optimized (html outline)
- [x] printability (print css)
- [x] doesn’t break anything else

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    });
+});
+
+// #############################################################################
+// #COMMANDS#
+gulp.task('watch', function () {
+    gulp.watch(PROJECT_PATTERNS.sass, ['sass']);
+});
+
+gulp.task('default', ['sass', 'watch']);
```
Comment (Brankub) [inline gulpfile.js]: please add last line

File: package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        "karma-fixture": "0.2.5",
+        "karma-html2js-preprocessor": "0.1.0",
+        "karma-jasmine": "0.3.6",
+        "karma-json-fixtures-preprocessor": "0.0.4",
+        "karma-phantomjs-launcher": "0.2.0",
+        "karma-sauce-launcher": "0.2.14",
+        "phantomjs": "1.9.17",
+        "protractor": "2.1.0"
+    }
+}
```
Comment (Brankub) [inline package.json]: missed last line. check .editorconfig

File: djangocms_admin_style/sass/libs/html5-boilerplate/_helpers.scss [hunk 3]
```diff
@@ -1,11 +1,14 @@
-@import "compass/typography/text/replacement";
-@import "compass/utilities/general/clearfix";
-
 //
 // Non-semantic helper classes
 // It's better to include these mixins in your own styles
 //
 
+@mixin hide-text () {
+	text-indent: -119988px;
```
Comment (dhorelik) [inline djangocms_admin_style/sass/libs/html5-boilerplate/_helpers.scss]: `-32767px` max value is considered safe

File: djangocms_admin_style/sass/libs/html5-boilerplate/_helpers.scss [hunk 4]
```diff
@@ -1,11 +1,14 @@
...
+	overflow: hidden;
+	text-align: left;
+}
+
 @mixin html5-boilerplate-helpers {
 	.ir { @include image-replacement; }
```
Comment (dhorelik) [inline djangocms_admin_style/sass/libs/html5-boilerplate/_helpers.scss]: tabs -> spaces ?

File: djangocms_admin_style/sass/partials/_content.scss [hunk 5]
```diff
@@ -27,7 +27,7 @@
 .inline-group {
 	border: 0; @include box(); padding: 10px 15px 15px;
 	.module {
-		background: none !important; @include border-radius(0); @include box-shadow(none); border: none !important;
+		background: none !important; border-radius: 0; box-shadow: none; border: none !important;
```
Comment (dhorelik) [inline djangocms_admin_style/sass/partials/_content.scss]: csscomb would help break the line here

File: djangocms_admin_style/sass/partials/_content.scss [hunk 6]
```diff
@@ -46,8 +46,8 @@
 	h3 { background: none; border-bottom: none; }
 }
 .submit-row {
-	@include border-radius(); border: 1px solid $line-color-light; padding: 10px;
-	@include box-shadow($background-color-white 0 0 $button-shadow-width inset !important);
+	border-radius: $default-border-radius; border: 1px solid $line-color-light; padding: 10px;
+	box-shadow: $background-color-white 0 0 $button-shadow-width inset !important;
 	background: $background-color-light;
```
Comment (dhorelik) [inline djangocms_admin_style/sass/partials/_content.scss]: this value will be overridden by the following line

File: djangocms_admin_style/sass/partials/_content.scss (hunk 5 above)
Comment (lory87) [inline djangocms_admin_style/sass/partials/_content.scss]: as it written on comment I will rund csscomb in separate pr :)

File: djangocms_admin_style/sass/libs/html5-boilerplate/_helpers.scss (hunk 4 above)
Comment (lory87) [inline djangocms_admin_style/sass/libs/html5-boilerplate/_helpers.scss]: Will run csscomb later and check then the spaces

File: djangocms_admin_style/sass/partials/_content.scss (hunk 5 above)
Comment (dhorelik) [inline djangocms_admin_style/sass/partials/_content.scss]: yep, missed that, sorry..

Comment (lory87) [inline djangocms_admin_style/sass/partials/_content.scss]: no problem :)

File: djangocms_admin_style/sass/partials/_messages.scss [hunk 7]
```diff
@@ -1,62 +1,62 @@
 /* Statuses Mixins */
 
 @mixin message-info {
-	@include background-image(linear-gradient( $info-color-light, $info-color 80%, $info-color-dark));
+	background-image: linear-gradient( $info-color-light, $info-color 80%, $info-color-dark);
 	.close {
-		@include box-shadow($info-color 0 0 2px inset);
+		box-shadow: $info-color 0 0 2px inset;
```
Comment (dhorelik) [inline djangocms_admin_style/sass/partials/_messages.scss]: inset should go first according to the spec - `box-shadow: inset 0 0 2px $info-color`

Comment (lory87) [inline djangocms_admin_style/sass/partials/_messages.scss]: will this be fixed if I run csscomb?

Comment (dhorelik) [inline djangocms_admin_style/sass/partials/_messages.scss]: ok

**Comment by dhorelik** [comment]
LGTM (needs csscomb though)

**Pull request merged**